lib/trot/template/slim.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Trot.Template.Slim do
+  @moduledoc """
+  Templating engine for handling ".slim" files using Calliope.
+  """
+
+  @behaviour Trot.Template.Engine
+
+  @doc false
+  def compile(file) do
+    quote do
+      unquote(file)
+      |> File.read!
+      |> Slime.Renderer.precompile 
+      |> EEx.compile_string assigns: var!(assigns)
```

Comment:
The `compile/1` function doesn't work. `EEx.compile_string/2` returns a quoted expression, which you're then double-quoting in the compile function. I think what you can do here is use `Slime.Renderer.render/2` in place of botht he Slime and EEx functions (and keep the `quote` block).

Reply:
That was the one I was having trouble with - I couldn't figure out where/how that function was ever called or used. Is there a test somehow/somewhere?

Reply:
I never figured out a good way to test this. Its used by the `Trot.Template` macros based on the mix environment - dev uses `compile/2` while other environments use `full_compile/2`. The difference is whether the template is precompiled in memory for faster responses or read off disk every time to allow for live reloading during development.